ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html

```diff
@@ -38,7 +38,7 @@
          data-sly-repeat="${accordion.exportedItems}"
          class="cmp-accordion__item"
          data-cmp-hook-accordion="item"
-         id="${accordion.exportedItems[item].id}"
+         id="${accordion.exportedItems[item].id}-item"
```

Comment:
this will lead to inconsistency between model and view

Reply:
Ensures every id attribute value used in ARIA and in labels is unique, it is not getting unique id.

Reply:
we have the accordion item id in the model also which is accordion.exportedItems[item].id
but here, markup will have a different id which may lead to errors.

Reply:
According to my analysis, there is no requirement of setting id for the accordion item markup. 
the id will be automatically set in the markup when we render the item resource through <div data-sly-resource>.
and that is the item for which view gets registered (and thus, the parent-child relationship). I think we can remove the id attribute from accordion item.
@sakshi-arora1 to confirm this understanding in this spl case of accordion.

Reply:
These things are used extensively in our product, we rely on main id to generate other ids such as:
```
id-label
id-shortDescription
id-longDescription
id-errorMessage
```
So if we add help-content, accessibility score is further reduced. Doing what is suggested in PR won't solve anything.
@pavi41 Kindly find out the reason of complain.
To solve this we need to get rid of all the ids and completely rely on classes/attribute inside the encapsulating element (having id).